main.js

```diff
@@ -68,8 +68,8 @@ function createWindow (autoLogin) {
     height: mainWindowState.height,
     x: mainWindowState.x,
     y: mainWindowState.y,
-    minWidth: 1000,
-    minHeight: 700,
+    minWidth: 636,
+    minHeight: 609,
```

Comment:
Interesting, but these look like magic numbers for me. It would be great to set them based on the research of popular screen sizes.

Reply:
good job

Reply:
I chose those numbers based on the minimum size that looks good for the way lepton is layed out (to fit languages, pinned, tags, etc).

Reply:
Interesting, let's merge and iterate on these numbers. Thanks :)

Reply:
:)